### ISSUE TITLE
SACD Iso Support Add-on conflicts with Video-Iso´s

### ISSUE THREAD

**Issue by Reemtsman** [xbmc/audiodecoder.sacd#14]
The SACD Iso Support Add-on does not only apply to SACD Iso Files. It also impacts other Video-Iso files like BD- or UHD-Isos used for the Movie Library.
 Importing of Iso Files to the Movie Library is not possible while the SACD Iso Support Addon is instaled an running. Movies are not found. When i go throug Kodi/Videos/Files to the Folder with the new movie, the iso file inside the folder is not displayed but the folder structure inside the iso.
Only after deinstallation of the SACD Iso Support Addon, the Iso filles are displayed in the folder again and the corresponding movie can be imported.
Please rework the Add-on so far that it effects exclusively the Iso Files of SACD´s connected to the Music Library. All other Video Iso files shall be unaffected by this Add-on. You can contact me if additional info is needed to fix this issue

**Comment by dabl2** [comment]
See [#11](https://github.com/xbmc/audiodecoder.sacd/issues/11)

**Comment by Reemtsman** [comment]
Dear @dabl2 : I overlooked thet #11 desribes the same problem. Sorry for the duplicate!

**Comment by dabl2** [comment]
Upon further investigation I do not believe this issue is in fact a duplicate of [#11](https://github.com/xbmc/audiodecoder.sacd/issues/11) although probably the same fix would take care of both.

With the plugin enabled many (perhaps all) of my .iso movies in my Kodi Movies library don't play, there is a quick flash and then nothing.

With the plugin disabled all good again.

**Comment by Reemtsman** [comment]
@dabl2 
The problem i wanted to describe here is pretty similar to the one reported in #11 , Based on that i would consider it a Duplicate. But it might as well be that the Problem has more than one rootcause.
In any case the SACD Iso Support Add-on is useless as long as it impacts Video ISO´s together with SACD ISO´s and needs a rework. Maybe by adding a limitation on the sources scanned into the Music Library,
